src/main/java/org/cascadebot/cascadebot/commands/fun/RandomNumberCommand.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.RandomUtils;
+import java.io.IOException;
+import java.util.Random;
+
+public class RandomNumberCommand implements ICommandMain {
+
+   
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length > 0) {
+            if (context.isArgInteger(0) == false) {
+                context.getTypedMessaging().replyDanger("Please provide numbers only");
+            } else {
+                int argOne = context.getArgAsInteger(0);
+                String randomNumber = RandomUtils.randomNumber(argOne).Integer.toString();
+                context.getTypedMessaging().replyInfo("Random number is `" + randomNumber + "`");
+            }
+        } else {
+             int randomNumberRaw = RandomUtils.randomNumber(10);
+             String randomNumber = Integer.toString(randomNumberRaw);
+             context.getTypedMessaging().replyInfo("Random number is `" + randomNumber + "`");
+        }
+    }
+
+    @Override
+    public String command() {
+        return "random";
```

Comment:
How about ;randnum with aliases of ;random and ;randomnumber and maybe ;number? I dunno, @binaryoverload?

Reply:
This was a decision by Ryan and I, I don't think it needs any aliases either

Reply:
Alrighty